day3OfJS/day3.js

```diff
@@ -0,0 +1,356 @@
+
+//vii. Find out the numbers of seconds elapsed from January 1, 1970 to now
+const numberSeconds = new Date();
+console.log(numberSeconds.getTime()); //1623882058236
+
+
+
+/*
+Exercises: Level 2
+1. Write a script that prompt the user to enter base and height of the triangle and calculate an area of a triangle (area = 0.5 x b x h).
+Enter base: 20
+Enter height: 10
+The area of the triangle is 50
+*/
+const base = Number(prompt('enter base of the triangle'));  //20
+const height = Number(prompt('enter height of the triangle'));  //10
+const area = 0.5 * base * height;
+console.log(`The area of the triangle is ${area}`);  //The area of the triangle is 100
+
+
+/*
+2. Write a script that prompt the user to enter side a, side b, and side c of the triangle and calculate the perimeter of triangle (perimeter = a + b + c)
+Enter side a: 5
+Enter side b: 4
+Enter side c: 3
+The perimeter of the triangle is 12
+*/
+const a = Number(prompt('enter fist value of the triangle'));
+const b = Number(prompt('enter second value of the triangle'));
+const c = Number(prompt('enter third value of the triangle'));
+const perimeter = a + b + c;
+console.log(`The perimeter of the triangle is ${perimeter}`); //The perimeter of the triangle is 12
+
+
+//3. Get length and width using prompt and calculate an area of rectangle (area = length x width and the perimeter of rectangle (perimeter = 2 x (length + width))
+const areaLength = Number(prompt('enter fist value of area rectangle'));   //first value 20
+const areaWidth = Number(prompt('enter second value of area rectangle'));  //second value 15
+const perimeterLength = Number(prompt('enter first value of the rectangle'));    //first value 30
+const perimeterWidth = Number(prompt('enter second value of the rectangle'));    //second value 20
+const areaRectangle = areaLength * areaWidth;
+const perimeterRectangle = 2 * (perimeterLength + perimeterWidth);
+console.log(`The area of a rectangle is ${areaRectangle} and the perimeter of the rectangle is ${perimeterRectangle}`); //The area of a rectangle is 300 and the perimeter of the rectangle is 100
+
+
+//4. Get radius using prompt and calculate the area of a circle (area = pi x r x r) and circumference of a circle(c = 2 x pi x r) where pi = 3.14.
+const r1 = Number(prompt('enter fist value of area circle'));   //first value 20
+const r2 = Number(prompt('enter second value of area circle')); //second value 10
+const r3 = Number(prompt('enter first value of the circle'));   //first value 25
+const pi = 3.14;
+const areaCircunference = pi * r1 * r2;
+const circunference = 2 * pi * r3;
+console.log(`The area of a circle is ${areaCircunference} and the circunference of the circle is ${circunference}`); //The area of a circle is 628 and the circunference of the circle is 157
+
+
+//5. Calculate the slope, x-intercept and y-intercept of y = 2x -2
+function slopeCalculate() {
+    const x = prompt("insert the intercept here", );
+    const y = prompt((2 * x) - 2);
```

Comment:
you need to calc `y `,`m`, `x` not to prompted to the user

Reply:
![image](https://user-images.githubusercontent.com/1114740/124207576-6d1fde80-daab-11eb-9aea-808e00510828.png)
the slope (m) is 2, the x= 1 and y = -2